ddtrace/internal/telemetry/writer.py

```diff
@@ -218,6 +221,16 @@ def __init__(self):
         # type: () -> None
         super(TelemetryMetricsWriter, self).__init__(interval=_get_telemetry_metrics_interval_or_default())
         self._namespace = MetricNamespace()
+        self._logs = []  # type: List[LogMetric]
+
+    def log_metric(self, level, message, stack_trace="", tags={}):
+        # type: (str, str, str, MetricTagType) -> None
+        """
+        Queues log metric
+        """
+        if config._telemetry_metrics_enabled:
+            with self._lock:
```

Comment:
Do we need a lock here. Adding to the logs queue should be an atomic operation.

Reply:
hey @mabdinur, what do you mean? there is a lock right now, thanks!

Reply:
I suspect this is not useful, append is already an atomic operation.
https://docs.python.org/3/faq/library.html#what-kinds-of-global-value-mutation-are-thread-safe